app/fn-guru/src/main/frontend/src/pages/app/exec-list/components/row-actions.tsx

```diff
@@ -28,7 +36,28 @@ export default function ({row}: Props) {
                 <DropdownMenuItem onClick={() => {
                     navigate(`/executions/${row.original.id}`)
                 }}>View</DropdownMenuItem>
+                {trigger && <TriggerSwitch triggerId={trigger.id} status={trigger.status}/>}
             </DropdownMenuContent>
         </DropdownMenu>
     )
 }
+
+
+type TriggerSwitchProps = { triggerId: string, status: string }
+const TriggerSwitch: FC<TriggerSwitchProps> = ({triggerId, status}) => {
+    const [setTriggerStatus, setTriggerRequested, loading, error] = useSetTriggerStatus()
+    const current = status === "Active"
+
+    function handleClick(e) {
+        e.stopPropagation()
```

Comment:
Why do we need this?

Reply:
The row itself had a listener that forwarded to the exec-detail page, the view button in the dropdown already provides this. I removed the row listener and the stopPropagation.